packages/platform/src/editor/Editor.tsx

```diff
@@ -80,24 +88,46 @@ function Placeholder(): JSX.Element {
  * @param props.logger - Logger instance.
  * @returns the editor element.
  */
-export default function Editor<TLogger extends LoggerBasic>({
-  usj,
-  viewOptions,
-  scrRef,
-  setScrRef,
-  nodeOptions,
-  isReadonly,
-  onChange,
-  logger,
-}: EditorProps<TLogger>): JSX.Element {
+const Editor = forwardRef(function Editor<TLogger extends LoggerBasic>(
+  {
+    defaultUsj,
```

Comment:
I might suggest wrapping this in a `memo` function and ignoring changes to `defaultUsj` to avoid extra renders, but it also isn't a big deal; feel free to ignore or make a note somewhere.

Reply:
Good suggestion. Like this?

Reply:
Sorry, I was talking about using the [`memo`](https://react.dev/reference/react/memo) function. You could make an `arePropsEqual` function that checks all the props and ignores the `defaultUsj` prop. But it really doesn't matter much and might just be a pre-optimization. Doing it like you had it before is fine.